### PR TITLE
Added overscroll-behavior property

### PR DESCRIPTION
**Pull request by yepninja**

### CODE REVIEW

**Comment by jonathantneal** [comment]
I think this is stage 0, because it has only made it as far as an Editor’s Draft hosted on csswg.org. Would you concur? If so, would you update the stage?

File: cssdb.json [hunk 1]
```diff
@@ -504,6 +504,15 @@
       }
     ]
   },
+  {
+    "id": "overscroll-behavior",
+    "title": "`overscroll-behavior` Property",
+    "description": "CSS property to control the behavior when the scroll position of a scroll container reaches the edge of the scrollport",
+    "specification": "https://drafts.csswg.org/css-overscroll-behavior",
+    "stage": 1,
```
Comment (jonathantneal) [inline cssdb.json]:
```suggestion
    "stage": 0,
```

See my other notes.

File: cssdb.json [hunk 2]
```diff
@@ -504,6 +504,15 @@
       }
     ]
   },
+  {
+    "id": "overscroll-behavior",
```
Comment (jonathantneal) [inline cssdb.json]: Could we change this to include the `property` prefix? We do this for other properties.

```suggestion
    "id": "overscroll-behavior-property",
```

File: cssdb.json [hunk 3]
```diff
@@ -504,6 +504,15 @@
...
+    "title": "`overscroll-behavior` Property",
+    "description": "CSS property to control the behavior when the scroll position of a scroll container reaches the edge of the scrollport",
```
Comment (jonathantneal) [inline cssdb.json]: Could we change this to drop the `CSS` explainer word and follow the `TYPE(S) for REASON(S)` style of other descriptions?

```suggestion
    "description": "Properties for controlling when the scroll position of a scroll container reaches the edge of a scrollport",
```

**Comment by yepninja** [comment]
Updated

**Pull request merged**